### ISSUE TITLE
OrderedDict type annotation is rejected

### ISSUE THREAD

**Issue by A-UNDERSCORE-D** [microsoft/pylance-release#151]
## Environment data

-   Language Server version: v2020.7.2
-   OS and version: Ubuntu 20.04
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.2
## Expected behaviour

Ordered Dict types from `typing` are understood and used for type checking in type annotations

## Actual behaviour

Warning: `Expected Class but received _Alias`. Type inferred to Any or other inference based on assignment

## Logs

```
[FG] parsing: /tmp/test.py (0ms)
[FG] binding: /tmp/test.py (1ms)
[BG] analyzing: /tmp/test.py ...
[BG]   parsing: /tmp/test.py (0ms)
[BG]   binding: /tmp/test.py (0ms)
[BG]   checking: /tmp/test.py (10ms)
[BG] analyzing: /tmp/test.py (11ms)
```

## Code Snippet / Additional information

```python
from typing import OrderedDict as odt, Any
from collections import OrderedDict

if __name__ == "__main__":
    test: odt[str, Any] = OrderedDict({"asd": None})

```

**Comment by erictraut** [comment]
Thanks for your bug report.

This was also reported (yesterday) in the Pyright repo, and it has already been addressed there. A fix will be included in the next release of Pylance.

https://github.com/microsoft/pyright/issues/864

**Comment by A-UNDERSCORE-D** [comment]
Ah alright, would you like me to leave this open or close it?

**Comment by erictraut** [comment]
You can leave it, and we'll close it after the release. Thanks!

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.7.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202073-21-july-2020
